Makefile

```diff
@@ -0,0 +1,44 @@
+.PHONY: help
+help: ## Print this help message and exit
+	@echo Usage:
+	@echo "  make [target]"
+	@echo
+	@echo Targets:
+	@awk -F ':|##' \
+		'/^[^\t].+?:.*?##/ {\
+			printf "  %-30s %s\n", $$1, $$NF \
+		 }' $(MAKEFILE_LIST)
+
+.PHONY: build
+build: ## Build the docker containers
+	docker-compose \
+		-f docker/docker-compose.yml \
+		build
+
+.PHONY: run-detach
```

Comment:
Can we have a straight `run` target where we get front and back in the same terminal??

Reply:
Sure, I'll add it